pallets/ah-migrator/src/staking/nom_pools.rs

```diff
@@ -141,3 +144,230 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 }
+
+fn normalize_generic_bonded_pool<T: Config>(
+	mut pool: GenericBondedPoolInner<BalanceOf<T>, T::AccountId, BlockNumberFor<T>>,
+) -> GenericBondedPoolInner<BalanceOf<T>, T::AccountId, BlockNumberFor<T>> {
+	use crate::Pallet;
+
+	if let Some(ref mut throttle_from) = pool.commission.throttle_from {
+		*throttle_from = Pallet::<T>::rc_to_ah_timepoint(*throttle_from);
+		// .saturating_add(One::one());
+	}
+	if let Some(ref mut change_rate) = pool.commission.change_rate.as_mut() {
+		change_rate.min_delay =
+			T::RcToAhDelay::convert(change_rate.min_delay).saturating_add(One::one());
```

Comment:
same here, ideally you need your own mapping, better even not looking into impl of `RcToAhDelay`, but from knowing what is expected. otherwise your test is:
``` rust
fn test_sum() {
  let expected_sum = sum(a + b);
  let actual_sum = sum(a + b);
  assert_eq(expected_sum, actual_sum);
}
```